### PR TITLE
/relay endpoint

### PR DESCRIPTION
**Pull request by sungo**
As per #144

### CODE REVIEW

File: Conch/lib/Conch/Model/Relay.pm [hunk 1]
```diff
@@ -108,6 +111,25 @@ sub connect_device_relay ( $self, $device_id, $relay_id ) {
...
+=head2 list
+
+Provide a list of all relays in the database as Class::Relay objects
+
+=cut
+
+sub list ( $self ) {
+	my @relays;
+	try {
+		for my $r ($self->pg->db->query("select * from relay;")->hashes->@*) {
```
Comment (lseppala) [inline Conch/lib/Conch/Model/Relay.pm]: You can use the SQL::Abstract interface, `...->db->select('relay')->...`

**Comment by daleghent** [comment]
I looked over the schema and this seems to be exactly what I need!

**Comment by sungo** [comment]
There are permissions tests in t/integration/04 to ensure that the readonly and integrator user classes can't access the relay list.

**Comment by lseppala** [comment]
Ah, I see it now. The way GH had folded the code, it looked unrelated.

**Pull request merged**